virttest/virt_vm.py

```diff
@@ -1190,6 +1190,19 @@ def print_guest_network_info():
             return self.wait_for_serial_login(timeout, internal_timeout,
                                               False, username, password)
 
+        # Attempt to log useful data
+        try:
```

Comment:
makes sense, can be handled as part of post-process instead?

Reply:
Well, handling it in post might be to late as the test might shut the machine down before the test finishes (this works even for intermittent VMs)